### DISCUSSION TITLE
Change in redis licensing terms

### DISCUSSION THREAD

**Discussion by ramkrish12** [goharbor/harbor#20356] [General]
Hi Experts, 

As we all aware of changes in Redis licensing from v7.4, is there any alternative plan to address the same ?

**Comment by MinerYang** [comment]
Hi @ramkrish12 ,

Thanks for bring it up and we are aware of this and @zyyw  will provide the following process

**Comment by zyyw** [comment]
Hi @ramkrish12 , Harbor only uses Redis - the software, as a cache component, instead of competing with Redis as an independent in-memory database product or service. It has no impact for Harbor to use Redis -the software, under the newly dual license - RSALv2 + SSPLv1.
